CloudFormationTemplate.yml

```diff
@@ -0,0 +1,205 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >
+  This CloudFormation template creates simple Lambda functions,
+  which prints CloudFormation resource Arn from the stack.  
+Resources:
+  LambdaFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+              - lambda.amazonaws.com
+          Action:
+            - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: LambdaFunctionPolicy
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
```

Comment:
Please split the policy with proper sid blocks:

Such as:
```yaml
          - Sid: CWAccess
            Effect: Allow
            Action:
              - logs:CreateLogStream
              - logs:PutLogEvents
            Resource:
              - !Sub arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/${LambdaFunctionName}:*
          - Sid: DynamDBAccess
            Effect: Allow
            Action:
              - dynamodb:scan
              - dynamodb:list
            Resource: 
            - !Sub arn:${AWS::Partition}:dynamodb:${AWS::Region}:${AWS::AccountId}:table/MetadataJson
          - Sid: S3Access
            Effect: Allow
            Action:
              - s3:ListAllMyBucket
              - s3:ListBucket
              - s3:PutObject
              - s3:PutObjectAcl
              - s3:GetBucketLocation
            Resource: 
            - !Sub arn:${AWS::Partition}:s3:::dbmetadata
            - !Sub arn:${AWS::Partition}:s3:::dbmetadata/*
```

And instead of having `logs:CreateLogGroup` action you can create the log group using cfn https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html

Reply:
and make use of proper pseudo parameters to avoid hardcoding
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html